physical_infra_provider_configuration/usage/topics/using_topology_widgets.adoc

```diff
@@ -1 +1,21 @@
-=== Using the Topology Widget for Physical Infrastructure Providers
\ No newline at end of file
+=== Using the Topology Widget for Physical Infrastructure Providers
+
+**
+The Topology widget extends CloudForms overview capabilities for Physical Infrastructure providers relationships with Physical Servers, Host and VMs. 
```

Comment:
This sentence probably shouldn't be in bold.

Reply:
Thanks, I've just updated.